US-CBP/cbp-theme
# Issue257

#### What's this PR do?
Tests the removal of the chokidar peerDependency within the package.json file.

#### Where should the reviewer start?
Maybe first look at the updates that were made to both the: "package.json" and "package-lock.json" files.

#### How should this be manually tested?
Check to see if BrowserSync works as expected in all 3 major browsers: Chrome, IE, Firefox

#### Any background context you want to provide?
This is in response to addressing the current issue #257 .

#### What are the relevant tickets?
n/a

#### Screenshots (if appropriate)
n/a

#### Questions:
- Is there a blog post?
#257 

- Does the knowledge base need an update?
not sure
